src/main/assembly/dist/cfg/application.properties

```diff
@@ -1,7 +1,64 @@
-bag-store.base-uri=http://localhost
-bag-store.uuid-component-sizes=2,30
+### General settings ###
+
+#
+# The base URL to use for local-file-uri's. This should normally be http://localhost.
+#
+bag-store.local-file-uri-base=http://localhost
+
+#
+# When transforming a UUID to the container path for a bag in the bag store, the slashes will
+# form path components with the sizes in the configured list. The numbers must therefore add
+# up to 32, which is the total length of a UUID when written out in hexadecimal (without dashes).
+#
+# Example: the pattern 8,8,16 will set up easy-bag-store to convert
+#
+#   59be6864-81ce-42cf-b671-1b8b2cf73705
+#
+# into
+#
+#   59be6864/81ce42cf/b6711b8b2cf73705
+#
+bag-store.uuid-slash-pattern=2,30
+
+#
+# The POSIX file permissions to set on directories (including the bag's base directory) and regular
+# files. The permissions should disallow any modification of the bags, but give enough privileges
+# for the bag store to operate. For example the "execute" permission on directories is necessary for
+# being able to list the contents of a directory.
+#
 bag-store.bag-file-permissions=r--r--r--
-staging.base-dir=/srv/dans.knaw.nl/stage
+bag-store.bag-dir-permissions=r-xr-xr-x
```

Comment:
Dir en file permissions uitgesplitst. Op een directory is `x` ook nodig, om de listing te kunnen bekijken.